content/admin/administration-panel/Employee Records.adoc

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
I am not sure this page is needed, we already have a good explanation here:

http://docs.suitecrm.com/admin/administration-panel/users/

which has the advantage of correctly explaining the not-very-intuitive relationship/difference between `user` and `employee` in SuiteCRM.

@likhobory what do you think?

Reply:
Agree, the required description may well be located in the /user/core-modules/Employees.adoc and /admin/administration-panel/Users.adoc files. Perhaps, I will remove the file Employee Records.ru.adoc and add some additional description to the existing files.